pkg/install/install.go

```diff
@@ -128,46 +160,46 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 	}
 
 	for {
-		i.log.Printf("starting phase %s", i.doc.OpenShiftCluster.Properties.Install.Phase)
 		switch i.doc.OpenShiftCluster.Properties.Install.Phase {
```

Comment:
just check if `steps[i.doc.OpenShiftCluster.Properties.Install.Phase] == nil` instead of the extra switch